python/src/algo_module.i

```diff
@@ -85,6 +85,7 @@
 %include KarhunenLoeveProjection.i
 %include KarhunenLoeveLifting.i
 %include KarhunenLoeveValidation.i
+%include CubaIntegration.i
```

Comment:
new classes must go to experimental_module.i instead (modify also python/src/CMakeLists.txt accordingly)